pylint/testutils/primer/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+__all__ = ["PackageToLint", "PRIMER_DIRECTORY_PATH"]
```

Comment:
The path to the primer directory to not need to be modified because it's relative (``PRIMER_DIRECTORY_PATH = Path("tests") / ".pylint_primer_tests"``)